suite-common/analytics/src/analyticsSlice.ts

```diff
@@ -0,0 +1,75 @@
+import { AnyAction, PayloadAction } from '@reduxjs/toolkit';
+import { memoize } from 'proxy-memoize';
+
+import { createSliceWithExtraDeps } from '@suite-common/redux-utils';
+
+type AnalyticsSliceState = {
+    sessionId?: string;
+    instanceId?: string;
+    enabled?: boolean;
+    confirmed?: boolean; // Confirmed through analytics consent?
```

Comment:
yes

Reply:
This was just comment to explain what it means - there was comment in previous reducer as well. I guess I can remove it as it's kind of self explanatory

Reply:
https://github.com/trezor/trezor-suite/pull/7622/commits/48d2e6372febcff336d648844ed47bc0f8784ca6